frontend/src/components/common/form/ParentSelect.tsx

```diff
@@ -21,8 +21,6 @@ interface IParentSelect {
   required?: boolean;
   /** determine whether this component is disabled or not */
   disabled?: boolean;
-  /** if this component is being used as a filter do not pre populate the value */
```

Comment:
Is this component not be used in the filterbar anymore?

Reply:
It is, I just rearranged the logic so you don't need it anymore